ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml

```diff
@@ -2,7 +2,7 @@ ref:
   as: openshift-qe-cluster-density-v2
   from_image:
     namespace: ci
-    name: ocp-qe-perfscale-ci
+    name: telco-bastion
```

Comment:
will this image work for both the telco and self-managed env?

Reply:
actually good call out - I will switch back to the QE image since we added ssh - thanks @paigerube14

Reply:
Updated. Thanks!